content/quiz_events.md

```diff
@@ -92,8 +92,9 @@ Coaches, inclding assistant coaches, of the teams participating in a quiz may pl
 ## Fouls
 - Fouls may be called by any quiz official:  the quizmaster, answer judge, or scorekeeper.
 - A foul must be called before the quizmaster calls for the next question.
-- If the officials fail to call a foul, a coach or team captain may call it to the attention of the quizmaster.  If in the quizmaster's (and/or the judge's) opinion a foul might have been committed, the quizmaster will give due warning to the violating team.  A foul may not be charged, and points deducted when the quizmaster receives such help from a coach or captain.
-- The quizzer upon whom a foul is called becomes ineligible to answer that numbered question, including the A and B questions.
+- If the officials fail to call a foul, a coach or captain may call it to the attention of the quizmaster.  If in the quizmaster's (and/or the other officials') opinion a foul might have been committed, the quizmaster will give due warning to the violating team.  A foul may not be charged, and points deducted when the quizmaster receives such help from a coach or captain.
```

Comment:
> A foul may not be charged,

Does this mean it's up to the quiz master to decide if a foul will be charged? May or may not? Or is this saying a foul cannot be charged. 

Without previous experience this line could be tricky for someone new to quizzing to understand.

Reply:
You're right, that language is not precise enough--I'd call it a rule book bug. I will fix.

Reply:
I've traditionally interpreted it as QMs have discretion, an interpretation I've traditionally disliked.

Reply:
I've interpreted it as "cannot." It absolutely says "may not", but the tone/flow of the entire section implies otherwise.

Reply:
My interpretation has been the same as Gryphon's though I would prefer Scott's. If you are sure, then a call shouldn't be made against a quizzer unless you can verify through an official or an official's resources (i.e. audio recording, etc)

Reply:
I mean, I'm not sure I've ever had anyone bring a foul to my attention..... so this almost never cames up. 

As a coach, I brought a missed foul to the attention of the QM once. QM's wife (who was a scorekeeper) immediately jumped to his defense. 

That was a fun quiz though-- we were losing by like 80 on question 17. Something like 4 errors by the leading team, plus some bonus questions and 3rd/4th by us, and we ended up winning by > 10 points.

Reply:
We need to go with something super clear and current practice for v1, but since this is so rare, I don't know what current practice is. I'd be OK with going with "cannot" even though that's not what I've been doing. I think it's so rare a case that as long as we're not inventing some new standard, we'll be FE.